src/libs/ReportUtils.ts

```diff
@@ -1061,26 +911,26 @@ function getIcons(report, personalDetails, defaultIcon = null, defaultName = '',
         const parentReportAction = ReportActionsUtils.getParentReportAction(report);
         const workspaceIcon = getWorkspaceIcon(report, policy);
         const memberIcon = {
-            source: UserUtils.getAvatar(lodashGet(personalDetails, [parentReportAction.actorAccountID, 'avatar']), parentReportAction.actorAccountID),
+            source: UserUtils.getAvatar(personalDetails?.[parentReportAction.actorAccountID ?? -1]?.avatar ?? '', parentReportAction.actorAccountID ?? -1),
```

Comment:
@kubabutkiewicz Can you answer this?

Reply:
we need to put some default value because `parentReportAction.actorAccountID` can be undefined and we can't put undefined as a key in object, thats why we need to put something , I am putting -1 because there is no chance that there will be -1 key in personalDetails collection